cmake/main/build_index_header_cmake.py

```diff
@@ -0,0 +1,80 @@
+#######################################################################
```

Comment:
This file should not be duplicated here. It should be called by reference or move CMake files up a directory

Reply:
Or at least extract the common pieces to a utility py file that is shared between the two py files.

Reply:
ok, I'll add a python file with the actual conversion and have both (build_index_header.py and build_index_header_cmake.py) call it.